### PR TITLE
Network Quality fixes 2

### PR DESCRIPTION
**Pull request by lakshyacis**
**Changes based on flake8 and pylint.**
Reduce line length| Suppressed warnings| Added docstrings.

### CODE REVIEW

File: src/network/socks4a.py [hunk 1]
```diff
@@ -1,6 +1,4 @@
 """
-src/network/socks4a.py
```
Comment (PeterSurda) [inline src/network/socks4a.py]: `SOCKS4a proxy module`

File: src/network/socks5.py [hunk 2]
```diff
@@ -1,7 +1,4 @@
 """
-src/network/socks5.py
-=====================
```
Comment (PeterSurda) [inline src/network/socks5.py]: `SOCKS5 proxy module`

File: src/network/stats.py [hunk 3]
```diff
@@ -1,6 +1,5 @@
 """
-src/network/stats.py
-====================
+Establishes and communicate with connection
```
Comment (PeterSurda) [inline src/network/stats.py]: `Network statistics`

File: src/network/tcp.py [hunk 4]
```diff
@@ -1,9 +1,7 @@
-# pylint: disable=too-many-ancestors
 """
-src/network/tcp.py
-==================
+Establish a connection before transmitting a data
```
Comment (PeterSurda) [inline src/network/tcp.py]: `TCP protocol handler`

File: src/network/udp.py [hunk 5]
```diff
@@ -1,13 +1,12 @@
 """
-src/network/udp.py
-==================
+Maintains connections between hosts
```
Comment (PeterSurda) [inline src/network/udp.py]: `UDP protocol handler`

File: src/network/proxy.py [hunk 6]
```diff
@@ -1,6 +1,5 @@
 """
-src/network/proxy.py
-====================
+Set proxy is avaiable otherwise exception
```
Comment (g1itch) [inline src/network/proxy.py]: if?

File: src/network/udp.py (hunk 5 above)
Comment (lakshyacis) [reply src/network/udp.py]: Okay. Thank you

File: src/network/proxy.py (hunk 6 above)
Comment (lakshyacis) [reply src/network/proxy.py]: Yes, Thank you :)

**Pull request merged**